lib/logitech_receiver/base_usb.py

```diff
@@ -164,6 +164,14 @@
     'isDevice': True
 }
 
+_wired_device = lambda product_id: {
```

Comment:
You are over-riding the usb_interface for all wired devices.  I don't think that this is correct.   But it may be that the G700s needs this interface.  So please try out the other interfaces.  If the G700s needs interface 1 then there will have to be a more sophisticated way of looking for them.